SearchEngines.yml

```diff
@@ -74,6 +82,13 @@ Aguea:
     params:
       - q
     backlink: 's.py?q={k}'
+Allaverksamheter:
+  - 
+    urls:
+      - www.allaverksamheter.se
```

Comment:
This does not seem to be a real search engine. For me it seems that it only has an internal site search. But I don't understand swedish, so maybe I'm wrong. What exactly is listed there? Business locations?